### PR TITLE
Lesson 16 Wiki Notes

### PR DESCRIPTION
**Pull request by gildedgardenia**

### CODE REVIEW

File: Class-16-(ORM-Part-1).md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Class 16: Introduction to Object-Relational Mapping
```
Comment (jimflores5) [inline Class-16-(ORM-Part-1).md:1]: Don't forget to add a link to these notes on the Home page.

File: Class-16-(ORM-Part-1).md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+## Large Group Time (Instructor)
+
+### Class 16 Topics That Require Careful Attention
+
+1. The students went through a number of different steps to get their Spring apps going and storing events and event categories. Review each setup step and why it matters.
+   1. Creating a new schema and add a new user in MySQL Workbench.
+   1. Add the necessary dependencies in IntelliJ.
+   1. Add the user info for the schema in MySQL into ``application.properties``.
+1. Review what an entity class is and what it becomes in a relational database.
+1. Review what a repository is and what it becomes in a relational database.
```
Comment (jimflores5) [inline Class-16-(ORM-Part-1).md]: How about *italics* for "entity class" and "repository"?

File: Class-16-(ORM-Part-1).md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+1. How to create a repository interface.
+
+## Announcements
+
+1. Assignment #4 is open and students can start working on it!
+1. Check with your course manager for any additional announcements.
+
+## Large Group Time (Instructor)
+
+### Class 16 Topics That Require Careful Attention
```
Comment (jimflores5) [inline Class-16-(ORM-Part-1).md:17]: I'd also add a note to review the relationships between Java classes/objects vs. database tables/rows.

File: Class-16-(ORM-Part-1).md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+   1. Add the necessary dependencies in IntelliJ.
+   1. Add the user info for the schema in MySQL into ``application.properties``.
+1. Review what an entity class is and what it becomes in a relational database.
+1. Review what a repository is and what it becomes in a relational database.
+1. Review the studio!
+
+## Small Group Time: Class 16 Studio (TA Notes)
+
+1. If students didn't finish the exercises, they can check out the ``add-persistent-category`` branch of their ``coding-events`` repo. If they do not have that branch in their repository, they need to fetch from the upstream and merge into their repository.
+1. An ``AbstractEntity`` is a new concept. This is a good chance to share some examples of why you  as a developer may use an ``AbstractEntity`` in your work.
```
Comment (jimflores5) [inline Class-16-(ORM-Part-1).md]: Comma additions.  "...you, as a developer, ..."

File: Class-16-(ORM-Part-1).md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+   1. Creating a new schema and add a new user in MySQL Workbench.
+   1. Add the necessary dependencies in IntelliJ.
+   1. Add the user info for the schema in MySQL into ``application.properties``.
+1. Review what an entity class is and what it becomes in a relational database.
+1. Review what a repository is and what it becomes in a relational database.
+1. Review the studio!
+
+## Small Group Time: Class 16 Studio (TA Notes)
+
+1. If students didn't finish the exercises, they can check out the ``add-persistent-category`` branch of their ``coding-events`` repo. If they do not have that branch in their repository, they need to fetch from the upstream and merge into their repository.
```
Comment (jimflores5) [inline Class-16-(ORM-Part-1).md:30]: For those who are fresh out of their own training, I'd add a reminder of the git syntax for fetching and merging from LC's upstream original.

**Review by jimflores5 — APPROVED** [review]
Sally, the notes look good (not even any spelling errors).  With a few minor changes, these should be good to merge.

File: Class-16-(ORM-Part-1).md (hunk 1 above)
Comment (gildedgardenia) [reply Class-16-(ORM-Part-1).md:1]: Done!

File: Class-16-(ORM-Part-1).md (hunk 2 above)
Comment (gildedgardenia) [reply Class-16-(ORM-Part-1).md]: Done! Thanks!

File: Class-16-(ORM-Part-1).md (hunk 3 above)
Comment (gildedgardenia) [reply Class-16-(ORM-Part-1).md:17]: Done!

File: Class-16-(ORM-Part-1).md (hunk 4 above)
Comment (gildedgardenia) [reply Class-16-(ORM-Part-1).md]: Done!

File: Class-16-(ORM-Part-1).md (hunk 5 above)
Comment (gildedgardenia) [reply Class-16-(ORM-Part-1).md:30]: Done!

**Pull request merged**